hazelcast-client/src/main/java/com/hazelcast/client/map/impl/nearcache/invalidation/ClientMapMetaDataFetcher.java

```diff
@@ -68,7 +68,7 @@ protected void extractAndPopulateResult(InternalCompletableFuture future, Result
         for (Member member : members) {
             Address address = member.getAddress();
             ClientMessage message = encodeRequest(names, address);
-            ClientInvocation invocation = new ClientInvocation(clientImpl, message, address);
+            ClientInvocation invocation = new ClientInvocation(clientImpl, message, null, address);
```

Comment:
You don't need to change; but what I often do to prevent passing non relevant parameters (especially if it happens so frequently) is to make use of a telescoping constructor.

So
```
Foo(String bla){
   this(bla, null);
}

Foo(String bla, String bli){

}
```

Unless the number of arguments becomes unwieldy and you get many arguments of the same type, telescoping constructors are one of the easiest solutions.